docker/entrypoint.sh

```diff
@@ -30,6 +30,23 @@ install_local_deps() {
     done
 }
 
+process_init_files() {
+    local f
+    for f; do
+        case "$f" in
+            *.sh)
+                if [ -x "$f" ]; then
```

Comment:
We're using bash in docker scripts. It's safe and preferred to use bash `[[` operator.